src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/ConnectionPool/HttpConnectionPool.cs

```diff
@@ -28,7 +28,6 @@ internal sealed partial class HttpConnectionPool : IDisposable
         public const int DefaultHttpPort = 80;
         public const int DefaultHttpsPort = 443;
 
-        private static readonly bool s_isWindows7Or2008R2 = GetIsWindows7Or2008R2();
```

Comment:
We can delete the helper too
https://github.com/dotnet/runtime/blob/5a54b6dbe8dca1751f6b2d1bb53a88c19fe941ee/src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/ConnectionPool/HttpConnectionPool.cs#L1030